src/main/java/blackjack/domain/card/Score.java

```diff
@@ -0,0 +1,71 @@
+package blackjack.domain.card;
+
+import java.util.Objects;
+
+public class Score implements Comparable<Score> {
+    private static final int BLACK_JACK = 21;
+    private final int score;
+
+    private Score(int score) {
+        if (score < 0) {
+            throw new IllegalArgumentException("점수는 음수가 될 수 없어요");
+        }
+        this.score = score;
+    }
+
+    public static Score of(int score) {
+        return new Score(score);
+    }
+
+    public Score add(int value) {
+        return new Score(value + score);
+    }
+
+    public Score subtract(int value) {
+        return new Score(score - value);
+    }
+
+    public boolean isBlackJack() {
+        return score == BLACK_JACK;
+    }
+
+    public boolean isBurst() {
+        return score > BLACK_JACK;
+    }
+
+    public boolean isHigherThan(Score counterpart) {
+        return compareTo(counterpart) > 0;
+    }
+
+    public boolean isSameAs(Score counterpart) {
+        return compareTo(counterpart) == 0;
+    }
+
+    @Override
+    public int compareTo(Score o) {
+        return score - o.score;
```

Comment:
compareTo 같은 경우 직접 연산을 해도 되긴 하지만 조금 위험해서요, 기본 타입의 경우는 Wrapper 클래스에서 비교 메서드를 제공하고 있습니다 :)
```suggestion
        return Integer.compare(score, o.score);
```